integration-test/dns-api-mesos/project/plugins.sbt

```diff
@@ -1,3 +1,3 @@
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.2")
 
-addSbtPlugin("com.dwijnand" % "sbt-dynver" % "2.0.0")
+addSbtPlugin("com.github.sbt" % "sbt-dynver" % "2.0.0")
```

Comment:
Curious that it didn't update the version here, but it did on the one below. 

Btw, this is seems to be a very old version. Is that really used? Looks more like a copy-and-paste